### DISCUSSION TITLE
Change the name of the global leaderboard to all-time leaderboard

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#1117] [Ideas] [answered]
#1095 

I thought it was a nice idea but he closed it so @Miodec what do you think about it?

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
okie i guess neither miodec nor anyoen cares about the idea on the issue or this discussion

so ima say it's donezo

**Reply by Miodec** [reply]
Looks like i missed this question, sorry. I might make a poll about it on Discord, see what people think.

**Reply by frogfrogfrogfrogfrog** [reply]
Ah sorry I honestly thought no-one cared about this discussion. Thanks for taking it into consideration
